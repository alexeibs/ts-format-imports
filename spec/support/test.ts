import {readFileSync} from "fs";
import * as ts from "typescript";
import {class1} from 'package1/module1';
import {class2} from 'package2/module3';
import {class3} from 'package1/module1';
import {class4} from './module4';

export function func() {
  let sum = 0;
  for (let i = 0; i < 10; ++i)
    sum += i;
  return sum;
}
