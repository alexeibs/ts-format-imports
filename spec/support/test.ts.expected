import * as ts from 'typescript';

import {class4} from './module4';

import {readFileSync} from 'fs';

import {class1} from 'package1/module1';
import {class3} from 'package1/module1';

import {class2} from 'package2/module3';

export function func() {
  let sum = 0;
  for (let i = 0; i < 10; ++i)
    sum += i;
  return sum;
}
